frontend/express/public/stylesheets/styles/blocks/_sidebar.scss

```diff
@@ -182,6 +183,20 @@ $bg-color: #24292e;
         background-position: center;
         display: inline-block;
     }
+    &__app-name-type {
```

Comment:
you can get rid of this completely and use the bulma classes instead directly on your html node.
for flex box related classes - https://bulma.io/documentation/helpers/flexbox-helpers/
for spacing - https://bulma.io/documentation/helpers/spacing-helpers/
p.s. dont forget to prepend **bu-** to the classes you use. Example - **is-flex** becomes **bu-is-flex**